addons/csi-aws-ebs/aws-csi.yaml

```diff
@@ -679,7 +679,7 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
-            - name: EC2_ENDPOINT
+            - name: AWS_EC2_ENDPOINT
```

Comment:
Is this intended? https://github.com/kubermatic/kubeone/tree/main/addons/csi-aws-ebs mentions removing AWS_ env vars.

Reply:
There's zero context on why we changed this in the first place. If tests are passing with `AWS_`, we'll just roll on with that and drop that comment.